src/main/webapp/app/shared/metis/posting-header/post-header/post-header.component.html

```diff
@@ -1,25 +1,39 @@
-<div class="row justify-content-between my-1">
-    <div class="col-auto pe-0">
-        <span class="posting-header header-author-date">
+<div class="d-flex justify-content-between my-1">
```

Comment:
Not part of this PR, but it would make sense to create a component for it. Since we use more or less the same code in several files.